frontend/src/components/MyApplicationItem/compounds/MissionDetail.tsx

```diff
@@ -0,0 +1,107 @@
+import classNames from "classnames";
+import { useMemo } from "react";
+import { Mission } from "../../../../types/domains/recruitments";
+import { MY_MISSION_TOOLTIP_MESSAGE } from "../../../constants/messages";
+import Tooltip from "../../@common/Tooltip/Tooltip";
+import styles from "../MyApplicationItem.module.css";
+
+const Container = ({ children }: { children: React.ReactNode }) => {
+  return <div className={styles["detail-container"]}>{children}</div>;
+};
+
+const TestResult = ({
+  testStatus,
+  passCount,
+  totalCount,
+  message,
+}: Pick<Mission["judgement"], "testStatus" | "passCount" | "totalCount" | "message">) => {
+  const resultText = useMemo(() => {
+    switch (testStatus) {
+      case "NONE":
+        return <span className={classNames(styles["test-result-score"])}>없음</span>;
+      case "PENDING":
+        return (
+          <span className={styles["pending-container"]}>
+            <span className={classNames(styles["test-result-score"], styles["pending"])}>
+              테스트 중
+            </span>
+          </span>
+        );
+      case "SUCCESS":
+        if (passCount === null || totalCount === null) {
+          throw new Error("예상하지 못한 응답입니다.");
+        }
+
+        return (
+          <span
+            className={classNames(
+              styles["test-result-score"],
+              passCount / totalCount === 1 ? styles["pass"] : styles["fail"]
+            )}
+          >{`${passCount} / ${totalCount}`}</span>
+        );
+      case "FAIL":
+        return (
+          <span className={classNames(styles["test-result-score"], styles["fail"])}>{message}</span>
+        );
+      default:
+        return "";
+    }
+  }, [testStatus]);
+
+  return (
+    <div className={styles["test-result-text"]}>
+      <span>예제 테스트 결과 : </span>
+      {resultText}
+    </div>
+  );
+};
+
+const PullRequestAddress = ({ children: pullRequestUrl }: { children: string | null }) => {
```

Comment:
c : 컴포넌트의 이름은 Address인데, props의 이름은 Url이네요. 코드상으로 봤을때 통일화해도 상관없을 것 같습니다. 

```suggestion
const PullRequestAddress = ({ children: address }: { children: string | null }) => {
```

Reply:
Url로 통일했습니다. 예리하네요